cirq/ops/common_gates.py

```diff
@@ -103,6 +103,16 @@ def _act_on_(self, args: Any):
                 tableau.xs[:, q] ^= tableau.zs[:, q]
             return True
 
+        if isinstance(args, clifford.ActOnStabilizerCHFormArgs):
+            if protocols.is_parameterized(self) or self.exponent % 0.5 != 0:
+                return NotImplemented
+            assert all(
+                gate._act_on_(args) for gate in  # type: ignore
+                [HPowGate(),
```

Comment:
You should be able to use `cirq.H` (except from the file it's defined in, not the global scope).

Reply:
Just used `HPowGate` to be consistent with `ZPowGate` (since importing pauli_gate for `Z` leads to circular dependency)

Reply:
Fixed it